modules/ossm-about-service-mesh-custom-resource-definitions.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+// * install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-service-mesh-custom-resource-definitions_{context}"]
+= About Service Mesh Custom Resource Definitions
+
+Installing the {SMProductName} Operator also installs Custom Resource Definitions (CRD) that administrators can use to configure {istio} for {SMProductShortName} installations. The {olm-first} installs two categories of CRDs: {sail-operator} CRDs and {istio} CRDs.
+
+{sail-operator} CRDs define custom resources for installing and maintaining the {istio} components required to operate a service mesh. These custom resources belong to the `sailoperator.io` API group and include the `Istio`, `IstioRevision`, `IstioCNI`, and `ZTunnel` resource kinds. For more information on how to configure these resources, see the `sailoperator.io` link:https://github.com/istio-ecosystem/sail-operator/blob/main/docs/api-reference/sailoperator.io.md[API reference] documentation.
+
+{istio} CRDs are associated with mesh configuration and service management. These CRDs define custom resources in several `istio.io` API groups, such as `networking.istio.io` and `security.istio.io`, and include various resource kinds, such as `AuthorizationPolicy`, `DestinationRule`, and `VirtualService`, that administrators use to configure a service mesh.
```

Comment:
I don't have a brilliant suggestion here but is there any way to break that second sentence into a couple that are less complex?

Reply:
The developer and i really wrestled with that. Looking at it now, I was able to break it into two sentences, but it's still a bit technical. We thought providing these details wold be helpful for our customers.

Reply:
yeah, the detail is good - just a long read to keep it all in one's head as one sentence :)